etc/ci/performance/test_all.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+export DISPLAY=:0 # In servo-linux1
```

Comment:
Also, please update the shebang and add robustness options at the top of the script as outlined in the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md).